lib/model-interfaces/langchain/functions/request-handler/adapters/sagemaker/mistralai/mistral_instruct.py

```diff
@@ -58,7 +58,7 @@ def get_llm(self, model_kwargs={}):
             model_kwargs=params,
             callbacks=[self.callback_handler],
         )
-    
+
     def get_qa_prompt(self):
         template = """<s>[INST] Use the following pieces of context to answer the question at the end. If you don't know the answer, just say that you don't know, don't try to make up an answer.[/INST]
```

Comment:
Not related to this PR, but it could `<s>` should be removed. Mistral prompts should be identical to LLama2 Chat SM prompts